packages/itwinui-css/src/menu/menu.scss

```
@@ -100,6 +100,26 @@ $iui-active-outline: thin solid var(--iui-color-border-accent);
   }
 
   &.iui-active {
+    @supports (selector(.iui-active:has(+ .iui-active))) {
```

this check doesn't need any specific selectors. should just be:
```suggestion
    @supports selector(:has(+ *)) {
```